website/config.js.example

```diff
@@ -0,0 +1,4 @@
+API_URL = 'http://localhost:5001/api/?';
+TILELAYER = 'https://{s}.tile.hosted.thunderforest.com/komoot-2/{z}/{x}/{y}.png';
```

Comment:
I guess most people have high definition we could use `https://{s}.tile.hosted.thunderforest.com/komoot-2/{z}/{x}/{y}@2x.png`; instead. But maybe you also know about an option in leaflet to automatically change the URL depending on the display density? I found [detectReting](https://leafletjs.com/reference-1.7.1.html#tilelayer-detectretina) but that seems to produce pretty small maps.

Reply:
I'll have a look. I wanted to update leaflet anyways. That's a good opportunity to check that as well.

Reply:
Use `{r}` and leaflet will add `@2x` for those displays which support high resolution https://leafletjs.com/reference-1.7.1.html